packages/firestore/src/core/sync_engine.ts

```diff
@@ -602,6 +609,59 @@ export class SyncEngine implements RemoteSyncer, SharedClientStateSyncer {
     }
   }
 
+  /**
+   * Registers a user callback with all the pending mutations at the moment of calling.
```

Comment:
Can you update this to:

Registers a user callback that resolves when all pending mutations at the moment of calling are acknowledged .

You can then remove the second sentence.

Reply:
Done.